README.md

```diff
@@ -36,55 +42,56 @@ The guidelines described below are based on:
 
 ## Directory Structure
 
-* Group files by the [bounded context](http://martinfowler.com/bliki/BoundedContext.html) they belong to. When a context (directory, for instance) grows to contain more than 15 files, start to consider creating a separate context by-type for them. Your threshold may be different, so adjust as needed.
+* Group files by the [bounded context](http://martinfowler.com/bliki/BoundedContext.html) they belong to. When a context (directory, for instance) grows to contain more than 15 files, start to consider creating a separate context by-type for them. Your threshold may be different, so adjust as needed:
```

Comment:
@mgechev 15 files is a lot, any reason for this particular number
usually what we have (files):
- component\directive
- html
- md
- spec
- e2e
more rarely
- config
- route
- service

7-8 files in sum